homeassistant/components/tado/repairs.py

```diff
@@ -26,7 +26,7 @@ def manage_water_heater_fallback_issue(
                 hass=hass,
                 domain=DOMAIN,
                 issue_id=f"{WATER_HEATER_FALLBACK_REPAIR}_{water_heater_name}",
-                is_fixable=False,
+                is_fixable=True,
```

Comment:
I think showing the fix button would send the wrong message here. Aim for this issue to notify user about the manual actions they must take. They must configure overlay options to their liking. Showing the fix button would imply an automated fix.